controllers/medusa/medusabackupjob_controller_test.go

```diff
@@ -488,6 +525,26 @@ func findDatacenterCondition(status *cassdcapi.CassandraDatacenterStatus, condTy
 	return nil
 }
 
+func deleteDatacenterPods(t *testing.T, f *framework.Framework, ctx context.Context, dcKey framework.ClusterKey, dc *cassdcapi.CassandraDatacenter) {
+	for i := int32(0); i < dc.Spec.Size; i++ {
```

Comment:
For future, you probably want to do ``for i := 0; i < int(dc.Spec.Size); i++ { }``. int32 is a weird type.